### PR TITLE
chore(normal-dungeon): added Spanish translation

### LINKED ISSUE #180: Spanish translation
Adding Spanish translation for [normal-dungeon](https://markdown-dungeon.netlify.app/english/normal-dungeon/begin-journey)

### PR DESCRIPTION
**Pull request by roramigator**
Please review, closes #180.

### CODE REVIEW

**Review by ming-tsai — CHANGES_REQUESTED** [review]
Hi @roramigator,
Nice work, but please add this dungeon on [here](https://github.com/roramigator/markdown-dungeon/blob/master/start-new-game.md)

**Comment by roramigator** [comment]
I did 👍

image

File: start-new-game.md [hunk 1]
```diff
@@ -2,12 +2,14 @@
...
-* [Save yourself from the viruses!](english/covid-19/begin-journey.md)
+- [Save yourself from the viruses!](english/covid-19/begin-journey.md)
 
-* [Eu escolho você, a dungeon normal](portuguese/normal-dungeon/begin-journey.md)
+- [Eu escolho você, a dungeon normal](portuguese/normal-dungeon/begin-journey.md)
 
-* [TIME to travel](english/time-travel/begin-journey.md)
+- [TIME to travel](english/time-travel/begin-journey.md)
+
+- [Yo te elijo, calabozo normal](spanish/normal-dungeon/begin-journey.md)
```
Comment (roramigator) [inline start-new-game.md:15]: #182 @ming-tsai the dungeon has been added to file.

**Comment by ming-tsai** [comment]
> I did 👍
> 
> image
> 
> @ming-tsai is this ok?

Sorry, I didn't see it

**Review by ming-tsai — APPROVED** [review]

**Pull request merged**